src/gui/cmdwidget.cpp

```diff
@@ -0,0 +1,61 @@
+#include "cmdwidget.h"
+
+namespace NeovimQt {
+
+CmdWidget::CmdWidget(ShellWidget *parent) {
+    setParent(parent);
```

Comment:
The setparent should not be needed if you call the parent constructor, just `:QTextEdit(parent)`, or am I missing something.

Reply:
My bad, I definitely can just call QTextEdit(parent)